finesse/hardware/__init__.py

```diff
@@ -53,11 +53,10 @@ def _send_temperatures() -> None:
         temperatures = _DEFAULT_TEMPS
 
     # Get time from the time source.
-    epoch_time = _try_get_time()
-    if epoch_time is None:
+    time = _try_get_time()
+    if time is None:
         # On failure, set time to the UNIX epoch.
-        epoch_time = 0.0
-    time = datetime.fromtimestamp(epoch_time)
+        time = datetime.fromtimestamp(0.0)
```

Comment:
Add this to `try_get_time` so it always returns a `datetime` object.

Reply:
+1. Then you can rename it to just `_get_time()`.